docs/typography/word-break.md

```diff
@@ -0,0 +1,25 @@
+# Word Break
+
+ZSS contains utility classes for setting values for the `word-break` property.
+
+## Source files
+
+- [utilities/_word-break.scss](../../src/utilities/_word-break.scss)
+
+## Variables
+
+Since the float CSS properties have a fixed set of values, there is no variable to configure.   
```

Comment:
Hier staat nog iets over de float CSS properties. Dit zou voldoende moeten zijn `There is no variable to configure`.

Reply:
Als er niks over valt te zeggen, hoeft er wat mij betreft geen kopje over variables in.